hedera-node/src/main/java/com/hedera/services/context/domain/security/PermissionFileUtils.java

```diff
@@ -72,6 +72,9 @@ public static String permissionFileKeyForQuery(Query query) {
 		permissionKeys.put(ConsensusUpdateTopic, "updateTopic");
 		permissionKeys.put(ConsensusDeleteTopic, "deleteTopic");
 		permissionKeys.put(ConsensusSubmitMessage, "submitMessage");
+		permissionKeys.put(ScheduleCreate, "scheduleCreate");
```

Comment:
I think that we should see how to cover the new `schedule` ops in the `PermissionFileUtilsTest`